### ISSUE TITLE
Errors when invoking yep

### ISSUE THREAD

**Issue by anntzer** [fabianp/yep#6]
```
$ python -myep -c profile.py # or -c -- profile.py, same result
￼Using local file /usr/bin/python.
Using local file /tmp/tmpjhe2lx83.
substr outside of string at /usr/bin/pprof line 3618.
Use of uninitialized value in string eq at /usr/bin/pprof line 3618.
substr outside of string at /usr/bin/pprof line 3620.
Use of uninitialized value in string eq at /usr/bin/pprof line 3620.
/tmp/tmpjhe2lx83: header size >= 2**16
```
```
$ python -myep -v profile.py
Using local file /usr/bin/python.
Using local file profile.py.prof.
substr outside of string at /usr/bin/pprof line 3618.
Use of uninitialized value in string eq at /usr/bin/pprof line 3618.
substr outside of string at /usr/bin/pprof line 3620.
Use of uninitialized value in string eq at /usr/bin/pprof line 3620.
profile.py.prof: header size >= 2**16
```
```
$ python -myep -v -- profile.py
Using local file /usr/bin/python.
Gathering CPU profile from http://profile.py.prof/pprof/profile?seconds=30 for 30 seconds to
  /home/antony/pprof/python.1430301840.profile.py.prof
Be patient...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0curl: (7) Failed to connect to profile.py.prof port 80: Connexion refusée
Failed to get profile: curl 'http://profile.py.prof/pprof/profile?seconds=30' > /home/antony/pprof/.tmp.python.1430301840.profile.py.prof: No such file or directory.
```

**Comment by colinhercus** [comment]
I had a similar problem (not Python) until I realised how stupid I was. I'd done

export CPUPROFILE=test.prof
export LD_PRELOAD="/usr/local/lib/libprofiler.so"
testprog ...
pprof --web `which testprog` test.prof

If I stopped after running 'testprog' the prof files wasn't empty but after pprof it was and pprof crashed with the substr outside of string error.

What I realised later was that by setting and exporting LD_PRELOAD that libprofiler.so was also loaded for pprof overwriting test.prof. 
You just need to ensure LD_PRELOAD is not set when you run pprof.
